bin/serverless.js

```diff
@@ -11,15 +11,21 @@
 const isMainModule = !EvalError.$serverlessCommandStartTime;
 if (isMainModule) EvalError.$serverlessCommandStartTime = process.hrtime();
 
-const nodeVersion = Number(process.version.split('.')[0].slice(1));
-const minimumSupportedVersion = 12;
+const nodeVersionMajor = Number(process.version.split('.')[0].slice(1));
```

Comment:
Question - should we use `semver` module for validation here or rather keep it the way I propose it here?